src/elm/Util/Maybe.elm

```diff
@@ -0,0 +1,17 @@
+module Util.Maybe exposing (flatMap, join)
+
+{-| flatMap combinator for Maybes
```

Comment:
Might want to add a module comment above so that this comment is "connected" to the function below.

Reply:
Aah. Det var derfor elm-format insisterte på to mellomrom der. Prøvde å fjerne de urovekkende mange ganger.